app/routes/events/view/sessions/list.js

```diff
@@ -54,7 +54,7 @@ export default Route.extend({
       filterOptions = [];
     }
     return this.modelFor('events.view').query('sessions', {
-      include      : 'event,speakers',
```

Comment:
About removing events from here:
- When this was included any request to update state of sessions included a empty relation to events and eventually request used to fail stating that event relation should have id. 
- I tried to figure out the actual problem and tried to set relation manually but that meant increased code complexity and logic.
- I realized that we do not actually use that event relation that e are fetching and when I removed this every request was successful since this didn't involved any relation to event.

Let me know if this is wrong. I will dig more deeper into the code and try to see what's the problem/